Xiaomi/Mi8/res/values/config.xml

```diff
@@ -103,13 +103,13 @@
     <fraction name="config_autoBrightnessAdjustmentMaxGamma">200.0%</fraction>
     <fraction name="config_maximumScreenDimRatio">29.999996%</fraction>
     <integer name="config_autoBrightnessBrighteningLightDebounce">2000</integer>
-    <integer name="config_autoBrightnessDarkeningLightDebounce">6000</integer>
+    <integer name="config_autoBrightnessDarkeningLightDebounce">2000</integer>
```

Comment:
This new value comes from stock ROM? Where did the old value come from?

Reply:
Same as the reply of last review.
`6000` froms Oreo based ROM, and `2000` froms Pie based ROM.